test_color_coding.py

```diff
@@ -0,0 +1,13 @@
+from color_code_identification import get_color_from_pair_number
+from color_code_identification import get_pair_number_from_color
+
+
+def test_number_to_pair(pair_number, expected_major_color, expected_minor_color):

```

Comment:
Think of test-names that include their expectation. Do you see any possibilities of more tests?